src/main/flight/gps_rescue.c

```diff
@@ -59,9 +59,9 @@ typedef enum {
     RESCUE_FLY_HOME,
     RESCUE_DESCENT,
     RESCUE_LANDING,
+    RESCUE_DO_NOTHING,
```

Comment:
`RESCUE_IDLE` and `RESCUE_DO_NOTHING` sound similar to me. I guess they are not, though? I am asking just out of interest.

Reply:
Sorry to be slow replying.
`RESCUE_IDLE` means, GPS Rescue code is inactive, and is waiting for a signal to become active.  In this time, if Mag is available, it used for heading display while GPS is enabled.
`RESCUE_DO_NOTHING` is an active flight mode within GPS rescue where the rescue has reached a sanity check and was switch initiated, to 'buy some time' for the user to reverse the switch, rather than disarming the quad instantly.  The quad levels out and sets throttle a bit below their hover value, and will disarm on an impact.